diff --git a/src/Makevars.win b/src/Makevars.win
index 6a7213d..1b54e38 100644
--- a/src/Makevars.win
+++ b/src/Makevars.win
@@ -1,3 +1,15 @@
-CXX_STD = CXX14
-PKG_CXXFLAGS = -I../inst/include
-PKG_LIBS = -L. -lz -lpthread -pthread
+VERSION = 2.2.3
+COMPILED_BY ?= gcc-4.6.3
+TARGET = lib$(subst gcc,,$(COMPILED_BY))$(R_ARCH)
+
+PKG_LIBS = \
+    -L$(LIBRARY_LIB) \
+
+winlibs:
+	mkdir -p ../inst
+	"${R_HOME}/bin${R_ARCH_BIN}/Rscript.exe" --vanilla "../tools/winlibs.R" $(VERSION)
+
+clean:
+	rm -f $(SHLIB) $(OBJECTS)
+
+.PHONY: all winlibs clean
