diff --git a/setup.py b/setup.py
index 3bd53ad..74aae67 100644
--- a/setup.py
+++ b/setup.py
@@ -31,19 +31,6 @@ def get_extensions():
     extra_compile_args = {'cxx': []}
     extra_link_args = []
 
-    if WITH_CUDA:
-        Extension = CUDAExtension
-        define_macros += [('WITH_CUDA', None)]
-        nvcc_flags = os.getenv('NVCC_FLAGS', '')
-        nvcc_flags = [] if nvcc_flags == '' else nvcc_flags.split(' ')
-        nvcc_flags += ['-arch=sm_35', '--expt-relaxed-constexpr']
-        extra_compile_args['nvcc'] = nvcc_flags
-
-        if sys.platform == 'win32':
-            extra_link_args += ['cusparse.lib']
-        else:
-            extra_link_args += ['-lcusparse', '-l', 'cusparse']
-
     extensions_dir = osp.join(osp.dirname(osp.abspath(__file__)), 'csrc')
     main_files = glob.glob(osp.join(extensions_dir, '*.cpp'))
     extensions = []
@@ -57,8 +44,6 @@ def get_extensions():
             sources += [path]
 
         path = osp.join(extensions_dir, 'cuda', f'{name}_cuda.cu')
-        if WITH_CUDA and osp.exists(path):
-            sources += [path]
 
         extension = Extension(
             'torch_sparse._' + name,
